crates/rpc/rpc-eth-api/Cargo.toml

```diff
@@ -0,0 +1,76 @@
+[package]
+name = "reth-rpc-eth-api"
+version.workspace = true
+edition.workspace = true
+rust-version.workspace = true
+license.workspace = true
+homepage.workspace = true
+repository.workspace = true
+description = "Reth RPC `eth_` API implementation"
+
+[lints]
+workspace = true
+
+[dependencies]
+# reth
+revm.workspace = true
+revm-inspectors = { workspace = true, features = ["js-tracer"] }
+revm-primitives = { workspace = true, features = ["dev"] }
+reth-errors.workspace = true
+reth-evm.workspace = true
+reth-metrics.workspace = true
+reth-network-api.workspace = true
+reth-primitives.workspace = true
+reth-provider.workspace = true
+reth-revm.workspace = true
+reth-rpc-server-types.workspace = true
+reth-rpc-types.workspace = true
+reth-rpc-types-compat.workspace = true
+reth-tasks = { workspace = true, features = ["rayon"] }
+reth-trie.workspace = true
+reth-transaction-pool.workspace = true
+reth-evm-optimism = { workspace = true, optional = true }
+reth-chainspec.workspace = true
+reth-execution-types.workspace = true
+
+# ethereum
+alloy-dyn-abi = { workspace = true, features = ["eip712"] }
+alloy-sol-types.workspace = true
+secp256k1.workspace = true
+
+# rpc
+jsonrpsee = { workspace = true, features = ["server", "macros"] }
+jsonrpsee-types = { workspace = true, optional = true }
+serde_json.workspace = true
+
+# async
+async-trait.workspace = true
+futures.workspace = true
+parking_lot.workspace = true
+tokio.workspace = true
+tokio-stream.workspace = true
+
+# misc
+auto_impl.workspace = true
+derive_more.workspace = true
+dyn-clone.workspace = true
+metrics.workspace = true
+rand.workspace = true
+schnellru.workspace = true
+serde.workspace = true
+thiserror.workspace = true
+tracing.workspace = true
+
+[dev-dependencies]
+reth-evm-ethereum.workspace = true
```

Comment:
needs txpool + reth-provider with test-utils

Reply:
then why does CI pass

Reply:
aah for the tests not triggered on pr I guess

Reply:
I suppose it exposes some trait extensions or smthg like that ?

Reply:
I think this is only an issue when cargo t inside the crate itself

Reply:
confirmed, yes. I think we have this with op too in some crates, where op code is transitively imported by import of other dep with op feature enabled.

Reply:
wonder if there is a way to test this in ci..